Samples/excel-shared-runtime-global-state/README.md

```diff
@@ -0,0 +1,117 @@
+---
+page_type: sample
+products:
+- office-excel
+- office-365
+languages:
+- typescript
+extensions:
+  contentType: samples
+  technologies:
+  - Add-ins
+  createdDate: 3/15/2020 1:25:00 PM
+description: "This sample shows a simple project using the shared runtime to shared global data with the ribbon, task pane, and custom functions."
+---
+
+# (preview) Share global data with a shared runtime
+
+## Summary
+
+This sample shows how to set up a basic project that uses the shared runtime. The shared runtime runs all parts of your Excel add-in (ribbon buttons, task pane, custom functions) in a single browser runtime. This makes it easy to shared data through local storage, or through global variables.
+
+![Screen shot of the add-in with ribbon buttons enabled and disabled](excel-shared-runtime-global.png)
+
+> **Note:** The features used in this sample are currently in preview and subject to change. They are not currently supported for use in production environments. To try the preview features, you will need to [join Office Insider](https://insider.office.com/join). A good way to try out preview features is by using an Office 365 subscription. If you don't already have an Office 365 subscription, you can get one by joining the [Office 365 Developer Program](https://developer.microsoft.com/office/dev-program).
+
+## Features
+
+- Share data globally with ribbon buttons, the task pane, and custom functions.
+- Use a provided manifest XML file to quick start a new project with a shared runtime.
+
+## Applies to
+
+-  Excel on Windows, Mac, and in a browser.
+
+## Prerequisites
+
+To try the preview features used by this sample, you will need to [join Office Insider](https://insider.office.com/join).
+
+Before running this sample, make sure you have installed a recent version of [npm](https://www.npmjs.com/get-npm) and [Node.js](https://nodejs.org/en/) on your computer. To check if you have already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+
+## Solution
+
+Solution | Author(s)
+---------|----------
+Office Add-in share global data with a shared runtime | Microsoft
+
+## Version history
+
+Version  | Date | Comments
+---------| -----| --------
+1.0 | 3-15-2020 | Initial release
+
+## Disclaimer
+
+**THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.**
```

Comment:
Not a fan of the all-caps, bolded text here. Please use our standard alerts to make this deprecation notice stand out. For example:
> [!IMPORTANT]
> Essential information required for user success.

Reply:
This is an old legal requirement, but I'm checking to see if we can drop or alter this to make it easier on the eyes.